test/extended/images/helper.go

```diff
@@ -775,6 +775,13 @@ func (c *CleanUpContainer) Run() {
 		}
 	}
 
+	out, err := c.OC.SetNamespace(metav1.NamespaceDefault).AsAdmin().
```

Comment:
Please add a commentary why is this necessary.

Reply:
And a TODO to replace this with removals of individual blobs used in the test case.

Reply:
@miminar Done